astropy/visualization/wcsaxes/core.py

```diff
@@ -460,17 +460,13 @@ def draw(self, renderer, **kwargs):
 
         self._drawn = True
 
-    # MATPLOTLIB_LT_30: The ``kwargs.pop('label', None)`` is to ensure
-    # compatibility with Matplotlib 2.x (which has label) and 3.x (which has
-    # xlabel). While these are meant to be a single positional argument,
+    # While these are meant to be a single positional argument,
     # Matplotlib internally sometimes specifies e.g. set_xlabel(xlabel=...).
 
     def set_xlabel(self, xlabel=None, labelpad=1, loc=None, **kwargs):
```

Comment:
Would seem that one could just write `set_xlabel(self, xlabel, labelpad=1, ...)` now and remove the test below altogether?

Reply:
I am not familiar with how matplotlib API, so would like to hear also from @astrofrog or @Cadair here.

Reply:
Ok, reverted and updated comment.